LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018, the ctrl_bps_parsl contributors on GitHub, https://github.com/LSSTDESC/ctrl_bps_parsl/graphs/contributors.
+Copyright (c) 2018, the ctrl_bps_parsl contributors on GitHub, https://github.com/lsst/ctrl_bps_parsl/graphs/contributors
```

Comment:
You can probably just say:
```
See the COPYRIGHT file for copyright information.
```
like we do elsewhere. I'm not entirely sure this license can be right given that BPS is GPL.

Reply:
The license is as inherited from the DESC gen3_workflow package.

Reply:
It's not compatible with BPS though so it can't be BSD license.

I'd like to convert all the middleware to BSD but that needs paperwork.